lib/fastMerge.js

```diff
@@ -0,0 +1,99 @@
+// Mostly copied from  https://medium.com/@lubaka.a/how-to-remove-lodash-performance-improvement-b306669ad0e1
+
+/**
+* Is the object Mergeable
+*
+* @param val
+* @returns {*|boolean}
+*/
```

Comment:
NAB: Can you please update the JSDocs so the next line has always a one whitespace at the start? 
```suggestion
 * Is the object Mergeable
 *
 * @param val
 * @returns {*|boolean}
 */
```